docs/src/test/kotlin/docs/user/gettingstarted/pom.xml

```
@@ -0,0 +1,181 @@
+// #pom-xml
+        <?xml version="1.0" encoding="UTF-8"?>
```

Just formatting, but expect this wasn't meant to be indented

```suggestion
<?xml version="1.0" encoding="UTF-8"?>
```